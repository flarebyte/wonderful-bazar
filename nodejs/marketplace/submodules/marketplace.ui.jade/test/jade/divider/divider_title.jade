include mixins/divider
ul
  mixin divider_title(everything.title)
