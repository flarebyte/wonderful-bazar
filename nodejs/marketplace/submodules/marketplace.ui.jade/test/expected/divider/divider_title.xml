<ul><li data-role="list-divider">everything</li></ul>